libkbfs/bare_root_metadata_v3.go

```diff
@@ -193,15 +188,6 @@ func (md *BareRootMetadataV3) IsValidRekeyRequest(
 		// Copy mismatch.
 		return false, nil
 	}
-	writerSigInfoEqual, err := kbfscodec.Equal(codec,
```

Comment:
Actually, looking a the server code, I don't think so. We call `IsValidAndSigned` on RMDS, which in this PR will also check the writer signature. The only thing we have to change is to call `IsValidAndSigned` first (which we should be doing anyway).

We can probably rip out the check in `IsValidRekeyRequest` for MDv2, too, as it would be redundant. What do you think?